### PR TITLE
Adds gaslift optimization support for multisegmented wells (MSW)

### PR DESCRIPTION
**Pull request by hakonhagland**
Adds gaslift optimization support for multisegmented wells (MSW).

### CODE REVIEW

**Comment by hakonhagland** [comment]
jenkins build this please

File: opm/simulators/wells/GasLiftSingleWell.hpp [hunk 1]
```diff
@@ -69,7 +61,7 @@ namespace Opm
         void setAlqMaxRate_(const GasLiftOpt::Well& well);
 
         const Simulator &ebos_simulator_;
-        const StdWell &std_well_;
+        const WellInterface<TypeTag> &std_well_;
     };
```
Comment (totto82) [inline opm/simulators/wells/GasLiftSingleWell.hpp:65]: Please rename to well_  
and also:
const WellInterfaceGeneric &getStdWell() const override { return std_well_; }
to getWell()
to reflect the change from stdWell well to wellInterface. 
I

**Review by totto82 — CHANGES_REQUESTED** [review]
Thanks. Both the extra debugging and the inclusion of MSW looks good. Can be merged when the well naming is fixed. It is a bit confusing to call it std_well_ when it can be both msw and stw.

**Comment by hakonhagland** [comment]
jenkins build this please

File: opm/simulators/wells/GasLiftSingleWell.hpp (hunk 1 above)
Comment (hakonhagland) [reply opm/simulators/wells/GasLiftSingleWell.hpp:65]: Good idea! See added commit.

**Pull request merged**